src/canopy/chat_engine/query_generator/last_message.py

```diff
@@ -0,0 +1,17 @@
+from typing import List
+
+from canopy.chat_engine.query_generator import QueryGenerator
+from canopy.models.data_models import Messages, Query
+
+
+class LastMessageQueryGenerator(QueryGenerator):
+
```

Comment:
Please add some docstring describing the behavior and propose of this class. Also it's better to explain in the generate function that we don't use the `max_prompt_tokens` arg

Reply:
Done